boards/zigduino/doc.txt

```diff
@@ -0,0 +1,80 @@
+/**
+ * @defgroup    boards_zigduino Zigduino
+ * @ingroup     boards
+ * @brief       Support for the Zigduino board
+
+# Hardware
+## Pinout
+
+![Zigduino](https://www.iot-lab.info/assets/images/docs/boards/zigduino/zigduino.png)
+[Pinout](https://static1.squarespace.com/static/511f4f0de4b09463c7605f13/t/5275c46be4b07e72f74c7438/1383449707749/zigduino-r2-pinout.pdf)
+
+@warning    Unlike on other ATmega MCUs, the GPIOs are not 5V tolerant.
+
+@note       The 5V pin cannot be used to power the board, as the board is not
+            equipped with an voltage regulator. The pin is therefore not
+            connected. But it can be used to pass 5V to shields, if connected
+            to a 5V supply voltage.
+
+## Board
+The Zigduino board is an Arduino-compatible microcontroller platform that integrates
+a SOC with a 8-Bit AVR MCU and a IEEE 802.15.4 radio chip (ATmega128RFA1).
+
+## MCU Details
+| MCU                           | ATmega128RFA1                     |
+|:------------------------------|:----------------------------------|
+| Family                        | ATmega                            |
+| Vendor                        | Atmel                             |
+| Package                       | QFN/MLF                           |
+| SRAM                          | 16KiB                             |
+| Flash                         | 128KiB                            |
+| EEPROM                        | 4KiB                              |
+| Core Frequency                | 8MHz (16MHz no power save mode)   |
+| Oscillators                   | 32.768 kHz & 16 MHz               |
+| Timer                         | 6 ( 2x8bit & 4x16bit )            |
+| Analog Comparator             | 1                                 |
+| ADCs                          | 1x 15 channel 6 to 12-bit         |
+| USARTs                        | 2                                 |
+| SPIs                          | 3 (1 SPI & 2 USART SPI)           |
+| I2Cs                          | 1 (called TWI)                    |
+| Vcc                           | 1.8V - 3.6V                       |
+| Datasheet / Reference Manual  | [Datasheet and Reference Manual](http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-8266-MCU_Wireless-ATmega128RFA1_Datasheet.pdf) |
+| Board Manual                  | [Product Page](http://www.logos-electro.com/store/zigduino-r2) |
+
+The MCU comes with a 2.4 GHz IEEE 802.15.4 radio that is compatible with the
+Atmel AT86RF23x line of transceivers with the only difference being that it is
+not being accessed over an SPI bus, but instead the radio registers are directly
+mapped into memory.
+
+# Flashing RIOT
+Flashing RIOT on the CoreRF is done USB-TTL adapter.
```

Comment:
CoreRF ? Should it be zigduino instead ? (and there's a `with` missing before `USB-TTL`)